spec/reflexes/products_reflex_spec.rb

```diff
@@ -88,6 +90,7 @@
       }.to change{ product_a.name }.to("Pommes")
         .and change{ variant_a1.display_name }.to("Large box")
         .and change{ variant_a1.sku }.to("POM-01")
+        .and change{ variant_a1.price }.to(10.25)
```

Comment:
I guess that the purpose of `to_currency` is to display `10.2` as `10.20`. And we have to blank the unit to not get a dollar sign in there.